examples/guides/catalog.ipynb

```diff
@@ -102,9 +102,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "up42-py",
+   "display_name": "up42-pyapi-dev",
    "language": "python",
-   "name": "up42-py"
+   "name": "up42-pyapi-dev"
```

Comment:
I guess this you should revert, always a mess with the jupyter notebooks

Reply:
oh, ok